src/mp/util.cpp

```diff
@@ -81,7 +81,7 @@ std::string LogEscape(const kj::StringTree& string)
     string.visit([&](const kj::ArrayPtr<const char>& piece) {
         if (result.size() > MAX_SIZE) return;
         for (char c : piece) {
-            if ('c' == '\\') {
```

Comment:
🤦

Reply:
Surprised there isn't a compiler warning for these kinds of things.